cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -365,6 +387,19 @@ private void performUpgrade() throws Exception {
     applicationLifecycleService.upgrade(true);
   }
 
+  private void performHBaseUpgrade() throws Exception {
+
+    System.setProperty(AbstractHBaseDataSetAdmin.SYSTEM_PROPERTY_FORCE_HBASE_UPGRADE, Boolean.TRUE.toString());
```

Comment:
This code is shared between HbaseUpgrade and regular upgrade. Maybe we can call the `performHBaseUpgrade()` from `performUpgrade()`. Otherwise the logic is in two places.